lib/actions/user.js

```diff
@@ -49,65 +49,63 @@ function getMiddlewareVariables (state) {
   }
 }
 
+/**
+ * Attempts to fetch the auth0 access token and store it in the redux state, under state.user.
+ * @param auth0 The auth0 context used to obtain the access token for subsequent middleware fetches.
+ */
+export function fetchAuth0Token (auth0) {
+  return async function (dispatch, getState) {
+    try {
+      const accessToken = await auth0.getAccessTokenSilently()
+
+      dispatch(setCurrentUser({ accessToken }))
+    } catch (error) {
+      // TODO: improve UI if there is an errror.
+      alert('Error obtaining an authorization token.')
+    }
+  }
+}
+
 /**
  * Attempts to fetch user preferences (or set initial values if the user is being created)
  * into the redux state, under state.user.
- * @param auth0 If provided, the auth0 login object used to initially obtain the auth0 access token
- *   for subsequent middleware fetches (and also to initialize new users from auth0 email and id).
- *   If absent, state.user.accessToken will be used for fetches.
+ * @param auth0 If provided, the auth0 login object used to initialize the default user object (with email and auth0 id).
  */
 export function fetchOrInitializeUser (auth0) {
```

Comment:
I think this arg should be changed to `auth0User`.

Reply:
How about `auth0Context`? Because the actual auth0 user is in `auth0.user`, which is passed to method `createNewUser`.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/227/commits/77a1a2c0f422a76331c75fbaea09579946a51692 to `auth0User`.